poly/joint_test.go

```diff
@@ -0,0 +1,99 @@
+package poly
+
+import (
+	"fmt"
+	"testing"
+)
+
+/////// TESTING ///////
+
+func TestReceiverAddDealer(t *testing.T) {
```

Comment:
This function runs the test for the Receiver struct for its method "AddDealer". So that should follow the guidelines right ?